src/webhooks/models/webhooksFormatter.js

```diff
@@ -42,6 +44,40 @@ function slackWebhookFormat(message, options) {
     };
 }
 
+function getThresholdTeamsSummary(state, { testName, score }) {
+    let icon = '&#x1F680;';
+    if (state === WEBHOOK_EVENT_TYPE_BENCHMARK_FAILED) {
+        icon = '&#x1F622;';
+    }
+    return `${icon} *Test ${testName} got a score of ${score.toFixed(1)}`;
+}
+
+function getThresholdTeamsSubtitle(state, { benchmarkThreshold, lastScores, aggregatedReport, score }) {
+    let resultText = 'above';
+    if (state === WEBHOOK_EVENT_TYPE_BENCHMARK_FAILED) {
+        resultText = 'below';
+    }
+    return `this is ${resultText} the threshold of ${benchmarkThreshold}. ${lastScores.length > 0 ? `last 3 scores are: ${lastScores.join()}` : 'no last score to show'}` +
+        `.*\n${statsFormatter.getStatsFormatted('aggregate', aggregatedReport, { score })}\n`;
+}
```

Comment:
I think slack and teams should share the same message format, maybe except for the emojis. but the text content should be the same, that would also mean that we can share the code between the two formats.

Reply:
I have consolidated the threshold message generation, and passed in a boolean variable to drive which icon to use.
Fixed in commit 6719dd2a4a5e2d612b2793d2536f54afec2b6477